docs/subnets/reference-cli-commands.md

```diff
@@ -208,7 +208,13 @@ avalanche subnet export [subnetName] [flags]
 
 ### Subnet Import
 
-The `subnet import` command imports a Subnet configuration from a file or a git repository.
+The `subnet import` command imports configurations into avalanche-cli.
+
+This command supports importing from a file created on another computer,
+or importing from subnets running public networks
+(e.g. created manually or with the deprecated subnet-cli)
+
+#### Import from a file
```

Comment:
```suggestion
#### Import From a File
```

Reply:
Actually titlecase.com suggests "Import from a File", which is what I have chosen.